db/redis.go

```diff
@@ -310,7 +327,6 @@ func (r *RedisDriver) InsertCpes(fetchType models.FetchType, cpes models.Fetched
 		"VendorProducts":           {},
 		"DeprecatedVendorProducts": {},
 		"DeprecatedCPEs":           {},
-		"Titles":                   {},
```

Comment:
Remove "Titles" from the comment above "newDeps :=" line?

Reply:
fixed https://github.com/vulsio/go-cpe-dictionary/compare/515af55b87e1d0e287d08824d5b5e1e69b452e0f..11a8f719f36ec2a80a1c6cd24be93dfa4d9a65c0